Submodels/MTSET_call.py

```
@@ -1,63 +1,114 @@
+            The path to the MTFLO executable.
+        analysis_name : str
+            The name of the analysis case.
+        """
 
-    def __init__(self, *args):
-        file_path, analysis_name = args
-        self.fpath = file_path
-        self.analysisName = analysis_name
+        # Input validation
+        if len(args) != 2:
+            raise ValueError("Expected exactly 2 arguments: file_path and analysis_name") from None
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)